lib/web_ui/lib/src/engine/html/render_vertices.dart

```diff
@@ -186,18 +143,20 @@ class _WebGlRenderer implements _GlRenderer {
     assert(positionsBuffer != null); // ignore: unnecessary_null_comparison
     gl.bindArrayBuffer(positionsBuffer);
     gl.bufferData(positions, gl.kStaticDraw);
+    Object? positionLoc = gl.getAttribLocation(glProgram.program, 'position');
     js_util.callMethod(
-        gl.glContext!, 'vertexAttribPointer', <dynamic>[0, 2, gl.kFloat, false, 0, 0]);
+        gl.glContext!, 'vertexAttribPointer', <dynamic>[
```

Comment:
nit: if we add a trailing comma the formatting will be:
```
gl.glContext!, 'vertexAttribPointer', <dynamic>[
    positionLoc, 2, gl.kFloat, false, 0, 0,
]);
```
(I'm told it's also in the style guide :) )

Reply:
Done.